q2_dada2/plugin_setup.py

```diff
@@ -136,21 +137,24 @@
                    'value will be discarded.'),
         'trunc_q': ('Reads are truncated at the first instance of a quality '
                     'score less than or equal to this value. If the resulting '
-                    'read is then shorter than `trunc_len`, it is discarded.'),
+                    'read is then shorter than `trunc_len_f` or `trunc_len_r` '
+                    '(depending on the direction of the read) it is '
+                    'discarded.'),
         'n_threads': ('The number of threads to use for multithreaded '
                       'processing. If 0 is provided, all available cores will '
                       'be used.'),
         'n_reads_learn': ('The number of reads to use when training the '
                           'error model. Smaller numbers will result in a '
                           'shorter run time but a less reliable error '
-                          'model.'),
+                          'model. If 0 is provided, all input reads will be '
+                          'used.'),
```

Comment:
`If 0 is provided, all input reads will be used from the first sample.`

https://github.com/qiime2/q2-dada2/pull/36#discussion_r98472656